### PR TITLE
Change color list/methods to LED* to match hardware labels

### PR DESCRIPTION
**Pull request by RaulTrombin**
This PR change the previous LEDs methods, to use board labels instead colors.

- [ ] https://github.com/bluerobotics/navigator-rs/issues/32

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -103,21 +103,21 @@ pub enum AdcChannel {
 
 /// Set of options to control navigator's LEDs.
 #[derive(Debug, Clone, Copy)]
-pub enum LedColor {
+pub enum UserLed {
     /// Attached to LED_1 though GPIO 24.
-    Blue,
+    LED1,
```
Comment (patrickelectric) [inline src/lib.rs:108]: Following the current code style, this should be `Led1`

File: src/lib.rs [hunk 2]
```diff
@@ -103,21 +103,21 @@ pub enum AdcChannel {
 
 /// Set of options to control navigator's LEDs.
 #[derive(Debug, Clone, Copy)]
-pub enum LedColor {
+pub enum UserLed {
     /// Attached to LED_1 though GPIO 24.
```
Comment (joaoantoniocardoso) [inline src/lib.rs:107]: Can you add the color name for each one of these LEDs?

**Review by patrickelectric — CHANGES_REQUESTED** [review]
Just a minor name change for consistency

File: src/lib.rs (hunk 2 above)
Comment (joaoantoniocardoso) [reply src/lib.rs:107]: and it's "through" there, not "though"

**Comment by RaulTrombin** [comment]
thks, all fix applied

File: src/lib.rs (hunk 2 above)
Comment (RaulTrombin) [reply src/lib.rs]: did it.

**Review by joaoantoniocardoso — APPROVED** [review]

**Comment by RaulTrombin** [comment]
@patrickelectric It's rebased

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**